src/selectors/host/selectors.js

```diff
@@ -7,4 +7,6 @@ export const getHostRole = hostMachine =>
   get(hostMachine, ['metadata', 'labels', 'machine.openshift.io/cluster-api-machine-role']);
 
 export const getHostMachineName = host => get(host, 'spec.machineRef.name');
+export const getHostBmcAddress = host => get(host, 'spec.bmc.address');
 export const getHostErrorMessage = host => get(host, 'status.errorMessage');
+export const getHostPoweredOn = host => _.get(host, 'status.poweredOn');
```

Comment:
`isHostPoweredOn` ? maybe also good to indicate default as false?

Reply:
Done